src/avs/AVSDetailsTVLTab.jsx

```diff
@@ -0,0 +1,274 @@
+import {
+  Spinner,
+  Table,
+  TableBody,
+  TableCell,
+  TableColumn,
+  TableHeader,
+  TableRow,
+  Image,
+  Skeleton,
+  Spacer,
+  Tooltip
+} from '@nextui-org/react';
+import React, { useMemo, useState } from 'react';
+import {
+  BEACON_STRATEGY,
+  EIGEN_STRATEGY,
+  STRATEGY_ASSET_MAPPING
+} from './helpers';
+import { useTailwindBreakpoint } from '../shared/useTailwindBreakpoint';
+import { formatUSD, formatETH, formatNumber } from '../shared/formatters';
+
+export default function AVSDetailsTVLTab({
+  totalTokens,
+  lst,
+  ethRate,
+  isAVSLoading
+}) {
+  return (
+    <>
+      {/*line chart*/}
+      <div className="bg-content1 border border-outline flex items-center justify-center h-[512px] p-4 rounded-lg w-full">
+        <Spinner color="primary" size="lg" />
+      </div>
+
+      {/*layout*/}
+      <div className="flex flex-col md:flex-row w-full h-full">
+        <div className="basis-1/2 mt-4">
+          <TokensBreakdownList
+            totalTokens={totalTokens}
+            isAVSLoading={isAVSLoading}
+            ethRate={ethRate}
+          />
+          <Spacer y={4} />
+          <LSTBreakdownList
+            lst={lst}
+            isAVSLoading={isAVSLoading}
+            ethRate={ethRate}
+          />
+        </div>
+        {/* treemap */}
+        <div className="basis-1/2 mt-4 md:ml-4">
```

Comment:
...and remove `mt` from here. Also, instead of `<Spacer/>,` it's better to use a flexbox and remove margins for the treemap.

```suggestion
      <div className="flex flex-col md:flex-row gap-4 w-full h-full">
        <div className="basis-1/2 flex flex-col gap-4">
          <TokensBreakdownList
            totalTokens={totalTokens}
            isAVSLoading={isAVSLoading}
            ethRate={ethRate}
          />
          <LSTBreakdownList
            lst={lst}
            isAVSLoading={isAVSLoading}
            ethRate={ethRate}
          />
        </div>
        {/* treemap */}
        <div className="basis-1/2">

```